CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* Enabled testifylint
```

Comment:
No need to add a line about build/CI changes
Runtime code of ydb-go-sdk has no changes